webextension/scripts/settings.js

```diff
@@ -146,6 +214,7 @@ function switchTabWindow() { $('input[type="radio"]').not(':checked').prop('chec
 
 function addDocs () {
   let docs = {
+    'private-mode': 'Extension will not support any feature!',
```

Comment:
It looks good!
But, when I am not logged in and I hover over the SPN button (Private Mode OFF), it should disable the SPN button and flipped button should say "Login to save", right? but it doesn't, instead it shows me the last save.

Reply:
And, should we change the tooltip text?  :-)

Reply:
Ok. Let me look into this . Thank you !